code/crates/starknet/host/src/streaming.rs

```diff
@@ -1,3 +1,5 @@
+/// The most of algorithm logic is explained in '/malachite/code/crates/starknet/host/src/actor.rs'
+/// Above 'on_received_proposal_part'
```

Comment:
Let's do it the other way, ie. add the explanation here and link to it from `actor.rs`

Reply:
Actually, let's just copy-paste the explanation here as well.